spring-integration-zip/src/test/java/org/springframework/integration/zip/transformer/UnZipTransformerTests.java

```diff
@@ -45,40 +34,74 @@
 import org.springframework.messaging.MessagingException;
 import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.zeroturnaround.zip.ZipException;
+import org.zeroturnaround.zip.ZipUtil;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Map;
+
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.instanceOf;
 
 /**
- *
  * @author Gunnar Hillert
  * @author Artem Bilan
- *
  * @since 1.0
- *
  */
 @RunWith(SpringJUnit4ClassRunner.class)
 @ContextConfiguration
 public class UnZipTransformerTests {
 
-	@Rule
-	public TemporaryFolder testFolder = new TemporaryFolder();
+    @Rule
+    public TemporaryFolder testFolder = new TemporaryFolder();
```

Comment:
Something is still wrong with your indents. Please, reconsider it in favor of tabs.